# L5: Nucleotides → DNA and RNA
[5 Nucleotides, DNA and RNA.pdf](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/7a5ffae1-b3b6-4933-a0fe-49356a86b2bb/5_Nucleotides_DNA_and_RNA.pdf)
###### tags: `MDSA20210 - molec`
>Chapter 6 Meisenberg and Simmons 2nd Ed.

## **Nucleotides:**

- energy-rich compounds
    - chemical signals
    - enzyme co-factors
- nucleic acids
    - DNA and RNA
    - polymers of nucleotides
- 3 components
    - nitrogenous base
    - sugar (ribose or deoxyribose)
    - phosphate
    
    ![](https://i.imgur.com/vt7JmoH.png =250x)


### **Bases:**

2 kinds of bases: purine (2 cyclic sections), pyrimidines (1 cyclic area)

- 2 purine bases
    - Adenine (A)
    - Guanine (G)
- 2 pyrimidine bases
    - Cytosine (C )
    - Thymine (T in DNA) or Uracil (U in RNA → T but no -CH3 methyl group)
- ![](https://i.imgur.com/ZtYXKai.png =350x)
    

### **Ribose:**

- sugar
    - carbons numbered:
        - 1',2',3',4',5'
            - 1': where base attaches
            - 3' and 5': where nucleotides polymerise 
        - ![](https://i.imgur.com/jgJeACV.png =200x)
            
- straight-chain or ring forms
    ![](https://i.imgur.com/LXN0wOq.png =200x)
  

### **Nucleosides:**

- nucleoside = base plus sugar
    - adenosine, cytidine, guanosine, thymidine, uridine
        
       ![](https://i.imgur.com/JS72y0q.png =250x)
      

### **Phosphate:**

- if R = H
    - phosphoric acid
- -OH usually ionized
    - lose H+
    - -O-
- Phosphate group: PO~4~
        ![](https://i.imgur.com/2oYSxZ0.png =100x)
     

## **DNA and RNA nucleotides:**

- base + sugar + phosphate
- ![](https://i.imgur.com/9fCCUrV.png =250x)
    - nucleoside phosphate
    - DNA and RNA: attach phosphate to 5' carbon
    - RNA:
        - adenosine monophosphate
        - cytidine monophosphate
        - guanosine monophosphate
        - uridine monophosphate
    - DNA
        - deoxyadenosine monophosphate
        - deoxycytidine monophosphate
        - deoxyguanosine monophosphate
        - deoxythymidine monophosphate
    - ![](https://i.imgur.com/qZoYgoE.png =300x)

### Nucleotides as energy carriers
ATP: adenosine triphosphate
![](https://i.imgur.com/ey0Klhf.png =250x)
- ADP, AMP
    - adenosine diphosphate
    - adenosine monophosphate 
- main energy exchange reactions in cells
    - ATP ⇌ ADP + PO4
    - ADP ⇌ AMP + PO4

### Nucleotides as enzyme cofactors
#### NAD and NADP
- nicotinamide adenine dinucleotide (NAD)
- nicotinamide adenine dinucleotide phosphate (NADP)
    - derived from niacin
- NAD or NADP, each has 2 forms:
    - oxidised: NAD+ and NADP+
    - reduced: NADH and NADPH

#### FAD and FMN:
- flavin adenine dinucleotide (FAD)
- flavin mononucleotide (FMN)
    - derived from riboflavin

#### Polymerise nucleotides
Nucleotides can be linked
- forms a strand of DNA or RNA
- phosphates linked to 2 sugars
- phosphodiester linkages
    - link
    - PO~4~ at 5' end to 3' OH of next nucleotide
- chain has polarity
    - distinct ends
        - 5’ end
        - 3’ end
    - usually “read” 5’ → 3’

## DNA 3D structure
### History, discovery
Crick and Watson in 1950s
- race with Linus Pauling to predict structure

Chargaff’s rules:
- different organisms have different amounts of DNA but the following is constant for all organisms in all cells
[A] = [T]
[G] = [C]

X-ray diffraction (Rosalind Franklin, Maurice Wilkins)
- used to predict double helix structure
- 2 periodicities
    - 3.4Å
    - 34Å

### Structure
#### Complementary base pairing
- A-T base pair: 2 H-bonds
- C-G base pair: 3 H-bonds
![](https://i.imgur.com/jHl1cxQ.png =350x)

#### Helical structure
- 2 antiparallel complementary strands in helical structure 
    - 5'→ 3' aligned with 3'→ 5'
![](https://i.imgur.com/1a68FUR.png =300x)
- 3.4Å/base pair
- 34A/major+minor groove
- 10 base pairs/turn 
    - 10-11 in aq solns

### DNA replication
1. helicase unwinds DNA
2. primase enzyme attaches RNA primers to DNA for DNA polymerases to attach to
3. **DNA polymerases** attach to RNA primers
    i) DNA polymerases add complementary nucleotides from primer to primer (forming okazaki fragments)
    ii) ligase enzyme joins okazaki fragments together
4. original strands rewind **(replication is semiconservative)**

### DNA denaturation
![](https://i.imgur.com/On9vlmD.png =300x)
- more AT pairs = lower temp needed to denature
- more CG pairs = higher temp needed to denature
- (due to number H-bonds... as H-bonds increase, more energy/higher temperature needed to break them apart/denature)

## RNA
Genetic material of RNA virus

Functional:
- e.g. translation machinery
    - rRNA (ribosomal RNA)
    - tRNA (transfer RNA)
    
Regulatory:
- control of gene expression
    - miRNA (microRNA)

Gene Expression:
- mRNA (messenger RNA)
- copy of 1 gene for translation by ribosomes

### RNA secondary structure
RNA single stranded
- can form base pairs internally
- many functional RNAs have secondary structure
    - mismatch pairs, loops, etc. 
    
![](https://i.imgur.com/dAlsPT6.png =200x)

## Genomes
Usually DNA
- RNA virus (e.g. HIV)
### Bacteria:
- 1 large circle of DNA
    - chromosome
- 500kb – 6mb
    - plus smaller circular plasmids
- 500-6000 genes
    - genes tightly packed
    - gap of 20-200 bases between genes 

### Humans
- mitochondria
    - circular DNA, 16kb
- nuclear DNA in linear chromosomes
    - Huge!
        - 3gb x2 total
    - 25k genes
        - huge spaces between genes
        - genes interrupted by introns

### Viruses
- can be RNA or DNA
- RNA examples:
    - HIV
    - influenza
    - foot and mouth
    - SARS

### Chromatin
Human chromosomes
- 23 pairs
- average length
    - 1.3 x 108 bp
    - 43mm
- need very tight packing
    - 5 orders of magnitude
- wrap DNA around histones
    - 5 histones: H1, H2A, H2B, H3, H4
    - rich in + charged amino acids
    - highly conserved in evolution



